Parsers/ASimWebSession/Parsers/vimWebSessionzScalerZIA.yaml

```diff
@@ -82,6 +82,7 @@ ParserQuery: |
     and (isnull(endtime) or TimeGenerated <= endtime) 
     and ((array_length(httpuseragent_has_any) == 0) or (RequestClientApplication has_any (httpuseragent_has_any)))
     and ((array_length(srcipaddr_has_any_prefix) == 0) or has_any_ipv4_prefix(SourceIP, srcipaddr_has_any_prefix))
+    and ((array_length(ipaddr_has_any_prefix) == 0) or has_any_ipv4_prefix(SourceIP, ipaddr_has_any_prefix) or has_any_ipv4_prefix(DestinationIP, ipaddr_has_any_prefix) )
```

Comment:
I wonder whether array_concat in the previous line would not be more efficient.

Reply:
what lists would you concatenate?